src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -62,7 +62,7 @@
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.15" />
     <PackageReference Include="Microsoft.AspNetCore.OData" Version="7.7.0" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.1.3" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.2.0-preview4.23342.2" />
```

Comment:
Why this version?

Reply:
SqlClient support was just recently merged.

https://github.com/dotnet/SqlClient/issues/19#issuecomment-1848577977

Reply:
I see